.env.example

```diff
@@ -117,6 +117,10 @@ OPENDAL_WASABI_BUCKET=<bucket>
 OPENDAL_WASABI_ENDPOINT=<endpoint>
 OPENDAL_WASABI_ACCESS_KEY_ID=<ak>
 OPENDAL_WASABI_SECRET_ACCESS_KEY=<sk>
+# atomicdata
+OPENDAL_ATOMICDATA_TEST=false
+OPENDAL_ATOMICDATA_ROOT=/path/to/dir
+OPENDAL_ATOMICDATA_ENDPOINT=https://atomicdata.dev
```

Comment:
Instead of writing data into atomicdata.dev, it would be better for us to start a new atomicdata instance for testing. This will prevent adding unnecessary burden on the maintainer.

cc @joepio, what do you think?

Reply:
I checked again and the current code in PR is only visiting atomicdata.dev to read class/property definitions, and real data is stored locally (this is default behavior of atomic_lib), as you mentioned in the other comment.

Reply:
> stored locally (this is default behavior of atomic_lib)

Ok, so my previous concern is not needed. However, the behavior is not my expection. I want our users can read data from an existing atomic data server instead of just locally.